rpcd/etc/openstack_deploy/user_rpco_variables_defaults.yml

```diff
@@ -136,3 +136,7 @@ secure_cluster: true
 # List of secure flags to set on for a pool (options for the list are nodelete, nopgchange, nosizechange - prevents deletion, pg from changing and size from changing respectively).
 secure_cluster_flags:
   - nodelete
+
+# We need to override the OpenStack upper constraint for Mitaka which is version 1.9.0, We should be able to remove this in Newton as the upper constraint moves to 2.3.0
+repo_build_upper_constraints_overrides:
+  - "elasticsearch>=2.0.0,<3.0.0"
```

Comment:
I based that off the recommendation from the elasticsearch maintainers:
https://pypi.python.org/pypi/elasticsearch/2.3.0

Happy to change to be a specific version.